compiler/codegen/symbol.cpp

```diff
@@ -1181,12 +1181,17 @@ void FnSymbol::codegenPrototype() {
     std::vector<const char *> argumentNames;
 
     int numArgs = 0;
+    std::vector<int> refArgs;
     for_formals(arg, this) {
       if (arg->hasFlag(FLAG_NO_CODEGEN))
         continue; // do not print locale argument, end count, dummy class
       argumentTypes.push_back(arg->codegenType().type);
       argumentNames.push_back(arg->cname);
+
+      arg->codegenType();
       numArgs++;
+      if(arg->intent & INTENT_REF)
+        refArgs.push_back(numArgs);
```

Comment:
Here, numArgs is 1-based, since this happens after we incremented it. I would expect LLVM argument indices to be 0-based... but it's not immediately obvious to me from the LLVM documentation. If in fact the are 1-based, could you add a comment to that effect here?